### PR TITLE
Allow working along with Rails 4 and Rails 5 [#127768687]

### PR DESCRIPTION
**Pull request by nc-lex**

### CODE REVIEW

**Comment by nc-trevorb** [comment]
👍 

@pattertom :eyes: please

File: lib/rack/tasks/request_log_readers.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@ class Rack::RequestLogger
       module TaskHelpers
 
         class << self
+          include Rake::DSL if defined?(Rake::DSL)
```
Comment (pattertom) [inline lib/rack/tasks/request_log_readers.rb]: what is this for?

Comment (nc-lex) [inline lib/rack/tasks/request_log_readers.rb]: New versions of rake give some warnings saying that "desc" "namespace" are be moved to Rake::DSL.

**Comment by nc-chandraf** [comment]
@pattertom @nc-trevorb Can this be merged? This seems to help with removing a bunch of DSL warning like below while I am upgrading LAC server to Ruby 2.4.1 and Rails 4.2.8:
```
WARNING: Global access to Rake DSL methods is deprecated.  Please include
    ...  Rake::DSL into classes and modules which use the Rake DSL methods.
WARNING: DSL method Module#desc called at /Users/chandraf/.rvm/gems/ruby-2.4.1/gems/rack-request_logger-0.0.6/lib/rack/tasks/request_log_readers.rb:94:in '<module:TaskHelpers>'
```

**Pull request merged**